spec/lib/vcr/cassette/migrator_spec.rb

```diff
@@ -114,13 +115,14 @@
   end if RUBY_PLATFORM == 'java'
 
   # Use syck on all rubies for consistent results...
-  before(:each) do
-    YAML::ENGINE.yamler = 'syck' if defined?(YAML::ENGINE)
-  end
-
-  after(:each) do
-    YAML::ENGINE.yamler = 'psych' if defined?(YAML::ENGINE)
-  end
+  around(:each) do |example|
+    YAML::ENGINE.yamler = 'syck'
+    begin
+      example.call
+    ensure
+      YAML::ENGINE.yamler = 'psych'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.